docs/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -413,3 +414,17 @@ performance. This is because they bypass the storage driver and do not incur
 any of the potential overheads introduced by thin provisioning and 
 copy-on-write. For this reason, you should place heavy write workloads on data 
 volumes.
+
+## OverlayFS compatibility
+To summarize the OverlayFS's aspect which is incompatible with other
+filesystems:
+
+- **RPMs and Yum**. OverlayFS only implements a subset of the POSIX standards. 
+This can result in certain OverlayFS operations breaking POSIX standards. One 
+such operation is the *copy-up* operation. Therefore, using `yum` inside of a 
+container on a Docker host using the `overlay`/`overlay2` storage drivers is 
+unlikely to work without implementing workarounds.
```

Comment:
Was this issue specifically related to the inode changing on a file which was open for read? I know this is not specifically related to this PR but may be a good chance to soften to the language, "unlikely to work" is no longer accurate.